### ISSUE TITLE
`juliarc.jl` and `static_julia()` not compatible with Julia 1.1.0

### ISSUE THREAD

**Issue by cihga39871** [JuliaLang/PackageCompiler.jl#169]
I cannot generate an executable file for my code under Julia v1.1.0. (I can do it under Julia v1.0.3).

Error log:
```julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.1.0 (2019-01-21)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using PackageCompiler

julia> static_julia("/home/xx/.julia/packages/PackageCompiler/jBqfm/examples/hello.jl"; builddir="lib", Release=true, executable=true)
Julia program file:
  "/home/xx/.julia/packages/PackageCompiler/jBqfm/examples/hello.jl"
C program file:
  "/home/xx/.julia/packages/PackageCompiler/jBqfm/examples/program.c"
Build directory:
  "/home/xx/projects/xx/lib"
ERROR: Unexpected format of "Base.julia_cmd()", you may be using an incompatible version of Julia
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] build_julia_cmd(::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::String, ::String, ::Nothing, ::Nothing, ::Nothing) at /home/xx/.julia/packages/PackageCompiler/jBqfm/src/static_julia.jl:213
 [3] build_object(::String, ::String, ::String, ::Bool, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::String, ::String, ::Nothing, ::Nothing, ::Nothing) at /home/xx/.julia/packages/PackageCompiler/jBqfm/src/static_julia.jl:239
 [4] #static_julia#1(::Nothing, ::Bool, ::Bool, ::String, ::Nothing, ::Nothing, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Nothing, ::Bool, ::Bool, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::Nothing, ::typeof(static_julia), ::String) at /home/xx/.julia/packages/PackageCompiler/jBqfm/src/static_julia.jl:172
 [5] (::getfield(PackageCompiler, Symbol("#kw##static_julia")))(::NamedTuple{(:builddir, :Release, :executable),Tuple{String,Bool,Bool}}, ::typeof(static_julia), ::String) at ./none:0
 [6] top-level scope at none:0

```

It may be a result of standard library update:
- `Base.julia_cmd` now propagates the `--inline=(yes|no)` flag.
(https://docs.julialang.org/en/v1/NEWS/#Standard-library-changes-1)

**Comment by akashkgarg** [comment]
I ran into this, I had to modify my static_julia.jl with the changes in : https://github.com/JuliaLang/PackageCompiler.jl/pull/167/

**Comment by SimonDanisch** [comment]
I think this should be fixed on master!

**Comment by lucatrv** [comment]
PR #167 was already merged to master, and release v0.5.2 includes it, I guess we need to wait a few days for the [Julia package list](https://pkg.julialang.org/) to sync.

**Comment by SimonDanisch** [comment]
ok let's close this then :)
